entries/b/bitrix24.ru.json

```
@@ -0,0 +1,18 @@
+{
+  "Битрикс24>": {
+    "domain": "bitrix24.ru",
+    "additional-domains": [
+      "bitrix24.net"
+    ],
```

You can just re-use the original icon here and remove `img/b/bitrix24.ru.svg`.
```suggestion
    ],
    "img": "bitrix24.com.svg",
```

---

Done.